include/boost/url/detail/config.hpp

```diff
@@ -84,4 +86,9 @@
 # error Unknown or unsupported architecture, please open an issue
 #endif
 
+#ifndef BOOST_URL_DOCS
+// this is private
```

Comment:
self-explanatory, since it is in detail/config.hpp (and also protected with the macro)